src/main/java/org/nuxeo/segment/io/web/SegmentIOScriptResource.java

```diff
@@ -87,7 +87,7 @@ protected Object buildScript(String login) {
 
         Map<String, Object> ctx = new HashMap<String, Object>();
         ctx.put("writeKey", segmentIO.getWriteKey());
-        ctx.put("debugMode", segmentIO.isDebugMode());
+        ctx.put("debugMode", Boolean.toString(segmentIO.isDebugMode()));
```

Comment:
`String.valueOf` is generally preferred over individual classes `toString` methods.